src/server/plugins/views.ts

```diff
@@ -72,7 +72,7 @@ export default {
       location: request?.app.location,
       sessionTimeout: config.sessionTimeout,
       skipTimeoutWarning: false,
-      serviceStartPage: config.serviceStartPage || '#',
+      serviceStartPage: config.serviceStartPage,
```

Comment:
Can we remove the `config.serviceStartPage` entirely? I can see it gets set as `const serviceStartPage = `/${this.model.basePath}` in the PageControllerBase.ts which is good (so the links work in the forms), but we don't need that to be customisable by config.

Reply:
Sure, I'll go ahead and remove it from confg too. I wasn't sure if it'd come in useful if/when we support running in "plugin" mode. Probably not.

Reply:
Resolved with d694109cf3aa892c7e38bea3098feafa3d2d2734

(I renamed serviceStartPage to serviceUrl as that better reflects what it is and matches GDS)